context/Web3Modal.tsx

```diff
@@ -20,18 +21,18 @@ const metadata = {
 };
 
 export const config = createConfig({
-  chains: [PUB_CHAIN],
+  chains: [PUB_CHAIN, mainnet],
   ssr: true,
   transports: {
     [PUB_CHAIN.id]: http(PUB_WEB3_ENDPOINT, { batch: true }),
+    [mainnet.id]: http(PUB_WEB3_ENDPOINT, { batch: true }),
   },
   connectors: [
     walletConnect({
       projectId: PUB_WALLET_CONNECT_PROJECT_ID,
       metadata,
       showQrModal: false,
     }),
-    injected({ shimDisconnect: true }),
     // coinbaseWallet({ appName: metadata.name, appLogoUrl: metadata.icons[0] }),
```

Comment:
Not enabled?

Reply:
Right now coinbaseWallet only works without SSR. If we enable it, it fulls the console with errors. 
We can enable it if we go to prod, but at least when developing it's quite annoying

Reply:
If we don't find a fix then we should consider rolling back enabling SSR

Reply:
The warning/error appears in general in Next, doesn't matter if SSR in wagmi or not.

To be clear, it does work. It just shows an error in the console because the backend fails to run that piece of the code.